components/sections/home-page/GitHubMock.tsx

```diff
@@ -78,10 +77,12 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
               <Image alt="Doodles" src={BackgroundStroke} />
             </div>
             <div className="relative largeTablet:left-8 largeTablet:w-4/5 largeTablet:top-8">
-                <img
-                  src={topFeature?.image as unknown as string}
-                  alt="True impact to the OSS community image"
-                />
+              <Image
+                src={topFeature?.image as unknown as string}
+                alt="True impact to the OSS community image"
```

Comment:
Shouldn’t have to add a role of ima as it’s already an image. The alt text should be enough but it sounds like the eslint rules currently require an aria-label as well. I can take a peek at that tomorrow, but for now, let’s get this in to fix the the invisible links.

Reply:
Put up a small change to this after testing locally.